CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ endif()
 option (BUILD_AGENT "Build EasyMesh agent" ON)
 option (BUILD_CONTROLLER "Build EasyMesh controller" ON)
 
+# Add global project cmake modules path
+list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
```

Comment:
Ideally, the same commit should also remove this from the four component CMakeLists.txt (and the commit should do only that), but that's just a nitpick.